snapcraft/plugins/ant.py

```diff
@@ -38,21 +53,52 @@
 
 class AntPlugin(snapcraft.plugins.jdk.JdkPlugin):
 
+    @classmethod
+    def schema(cls):
+        schema = super().schema()
+        schema['properties']['ant-properties'] = {
+            'type': 'object',
+            'default': {},
+        }
+        schema['properties']['ant-build-target'] = {
+            'type': 'string',
+        }
+        schema['properties']['ant-dest-property'] = {
```

Comment:
is this need only because we don't support using something like `$SNAPCRAFT_PART_INSTALLDIR` in the yaml?